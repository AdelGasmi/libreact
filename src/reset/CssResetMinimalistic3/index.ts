import createCssResetComponent from '../createCssResetComponent';

export const css = {
  '*': {
      pad: 0,
      mar: 0,
      bd: 0,
      out: 0,
  },
};

const CssResetMinimalistic3 = createCssResetComponent(css);

export default CssResetMinimalistic3;
