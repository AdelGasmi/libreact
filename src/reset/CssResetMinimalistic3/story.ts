import {createElement as h} from 'react';
import {storiesOf} from '@storybook/react';
import {action} from '@storybook/addon-actions';
import {linkTo} from '@storybook/addon-links';
import CssResetMinimalistic3 from '.';
import ShowDocs from '../../../.storybook/ShowDocs'

storiesOf('CSS resets/CssResetMinimalistic3', module)
  .add('Documentation', () => h(ShowDocs, {name: 'reset/CssResetMinimalistic3'}))
  .add('Example', () =>
    h(CssResetMinimalistic3)
  );
