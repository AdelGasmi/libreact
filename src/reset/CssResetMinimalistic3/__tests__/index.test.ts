import {createElement as h} from 'react';
import {renderToStaticMarkup} from 'react-dom/server';
import CssResetMinimalistic3 from '..';

describe('<CssResetMinimalistic3>', () => {
  it('renders as expected', () => {
    const html = renderToStaticMarkup(h(CssResetMinimalistic3));

    expect(html).toMatchSnapshot();
  });
});
