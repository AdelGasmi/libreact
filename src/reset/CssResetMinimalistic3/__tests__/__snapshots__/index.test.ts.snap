// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CssResetMinimalistic3> renders as expected 1`] = `"<style>*{padding:0;margin:0;border:0;outline:0;}</style>"`;
